src/msg/async/AsyncConnection.cc

```diff
@@ -2364,12 +2358,19 @@ ssize_t AsyncConnection::write_message(Message *m, bufferlist& bl)
                              << " data=" << header.data_len
                              << " off " << header.data_off << dendl;
 
-  complete_bl.claim_append(bl);
+  if (bl.length() <= 256) {
```

Comment:
I agree with this idea if buffer length is small we can copy it. I think this idea is valuable to apply to upper layer like encoding/decoding. But here we will do nothing for bufferlist later except iterator this for sendmsg. sendmsg will copy these framgement memory in kernel side.

So do we have clear motivation to do this?